src/store/index.tsx

```diff
@@ -75,6 +79,7 @@ export const DerivProvider = ({ children }: DerivProviderProps) => {
                 deriv_api,
                 show_non_eu_popup,
                 setShowNonEuPopup,
+                breakpoints,
```

Comment:
@michio-rework Why would we need to keep this in the context? 🤔

Reply:
we're gonna have a separate `Context` for this one, but for this step we're gonna use the `DerivStore` for now. 
the reason behind the using `context` is mentioned in above comments.